### PR TITLE
minor fix to support safari

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

File: frontend/pulse3d/src/components/layouts/DashboardLayout.js [hunk 1]
```diff
@@ -20,16 +20,24 @@ export default function DashboardLayout({ children }) {
 
   useEffect(() => {
```
Comment (tannermpeterson) [inline frontend/pulse3d/src/components/layouts/DashboardLayout.js:21]: Is this `useEffect necessary anymore now that the one beneath it got added?

**Review by tannermpeterson — APPROVED** [review]

File: frontend/pulse3d/src/components/layouts/DashboardLayout.js (hunk 1 above)
Comment (luciipak) [reply frontend/pulse3d/src/components/layouts/DashboardLayout.js:21]: Yeah I'm trying to think of a better way of doing it, but I need it to get called everything the component mounts, when the account type changes, and when the router changes.

Comment (luciipak) [reply frontend/pulse3d/src/components/layouts/DashboardLayout.js:21]: because if a user refreshes the page, then it needs to happen on mount too

Comment (tannermpeterson) [reply frontend/pulse3d/src/components/layouts/DashboardLayout.js:21]: Gotcha

**Pull request merged**